components/observations/SimpleForm.tsx

```diff
@@ -63,10 +63,10 @@ export const SimpleForm: React.FC<{
 
   const {nationalAvalancheCenterHost} = React.useContext<ClientProps>(ClientContext);
 
-  const mutation = useMutation<Observation, AxiosError, Partial<ObservationFormData>>({
-    mutationFn: async (observationFormData: Partial<ObservationFormData>) => {
+  const mutation = useMutation<void, AxiosError, ObservationFormData>({
```

Comment:
`submitObservation` doesn't return an `Observation` now...but to be fair, it never did before either. remember this comment? https://github.com/NWACus/avy/blob/4339d9ba7d5443630a58285bcdb86a64784912c4/components/observations/submitObservation.ts#L143-L147